autogpt/setup.py

```diff
@@ -62,6 +64,8 @@ def prompt_user() -> AIConfig:
     else:
         try:
             return generate_aiconfig_automatic(user_desire)
+        except CriticalException:
```

Comment:
We should write something to the log here?

Reply:
I dont think so. We are throwing our own custom exception -> we know exactly what happened. Potentially might have it in the global exception handler, integrated with Sentry.